exporter/v1_compatibility.go

```diff
@@ -805,28 +805,38 @@ func specialMetrics(ctx context.Context, client *mongo.Client, m bson.M, l *logr
 	return metrics
 }
 
-func retrieveMongoDBEdition(ctx context.Context, client *mongo.Client, l *logrus.Logger) (string, error) {
+func retrieveMongoDBBuildInfo(ctx context.Context, client *mongo.Client, l *logrus.Logger) (buildInfo, error) {
 	buildInfoCmd := bson.D{bson.E{Key: "buildInfo", Value: 1}}
 	res := client.Database("admin").RunCommand(ctx, buildInfoCmd)
 
 	var buildInfoDoc bson.M
 	err := res.Decode(&buildInfoDoc)
 	if err != nil {
-		return "", errors.Wrap(err, "Failed to run buildInfo command")
+		return buildInfo{}, errors.Wrap(err, "Failed to run buildInfo command")
 	}
 
 	var edition string
 	modules, ok := buildInfoDoc["modules"].(bson.A)
 	if !ok {
-		return "", errors.Wrap(err, "Failed to cast module information variable")
+		return buildInfo{}, errors.Wrap(err, "Failed to cast module information variable")
 	}
+
+	var bi buildInfo
 	if len(modules) > 0 && modules[0].(string) == "enterprise" {
-		edition = "Enterprise"
+		bi.Edition = "Enterprise"
 	} else {
-		edition = "Community"
+		bi.Edition = "Community"
 	}
 	l.Debug("MongoDB edition: ", edition)
-	return edition, nil
+
+	_, ok = buildInfoDoc["psmdbVersion"]
+	if ok {
+		bi.Vendor = "Percona Mongo"
```

Comment:
`Percona Mongo` and `Mongo` are bad choices for vendor names. We want to differentiate between companies who provide MongoDB installations.

Alternatives that would be beter:
-  `Percona` and `MongoDB`
-  `Percona LLC` and `MongoDB Inc`